### PR TITLE
Content changes

### PR DESCRIPTION
**Pull request by MeghanHughes**
A bunch of content edits and additions of resource links.
Question - what do we think of using the NZ Herald image for the icon in the News section? It would have to be a little bigger. Don't know if I like it or not, but if we do we will have to find icons for each of the media outlets to be consistent. Otherwise I will revert the icons back to the generic ones.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -349,9 +354,9 @@ <h2>Project related news</h2>
           <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4">
             <div class="news-wrapper">
               <img src="img/website.png" alt="news icon" class="news-icon" />
-              <p class="news-source">Law Commission</p>
-              <p class="news-article"><a href="http://www.lawcom.govt.nz/news/relationship-property-review-underway">Relationship property review underway</a></p>
-              <p class="news-date">27 November 2017</p>
+              <p class="news-source">NZ Herald</p>
+              <p class="news-article"><a href="http://www.nzherald.co.nz/business/news/article.cfm?c_id=3&objectid=10870095">Diana Clement: Splitting property not always straightforward.</a></p>
+              <p class="news-date">9 March 2013</p>
```
Comment (janie-t) [inline index.html]: Put the articles in order of publication date

**Review by janie-t — COMMENTED** [review]
Put the articles on the site in order of publication date

File: index.html [hunk 2]
```diff
@@ -362,7 +367,62 @@ <h2>Project related news</h2>
               <p class="news-date">13 October 2017</p>
             </div>
           </div>
-
+        </div>
+        <div class="row">
+          <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4">
+            <div class="news-wrapper">
+              <img src="img/news.png" alt="news icon" class="news-icon" />
```
Comment (janie-t) [inline index.html]: Which is a news icon and which is a website?  News items for Herald, Scoop, Stuff, ODT. Website item for the law commission website.

File: index.html [hunk 3]
```diff
@@ -272,26 +273,29 @@ <h2>Resources</h2>
...
-              mention trusts at all. Some CAB’s offer free half hour sessions with a lawyer.</h4>
+            <h4>Some CABs offer free half hour sessions with a lawyer.</h4>
               <a class="btn btn-primary" a href="http://www.cab.org.nz/Pages/home.aspx">Visit CAB</a>
             </div>
             <!-- <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4"> -->
             <div class="resource">
               <h3>Legal Aid</h3>
               <h4>Visit the Department of Justice website for information on your eligibility for legal aid support.</h4>
-              <a class="btn btn-primary" a href="https://www.justice.govt.nz/courts/going-to-court/legal-aid/">Visit website</a>
+              <a class="btn btn-primary" a href="https://www.justice.govt.nz/courts/going-to-court/legal-aid">Visit website</a>
```
Comment (anna-marie-vos) [inline index.html]: Heya, Remove the "a" before the href. For all the hrefs. 

Also the colours on the buttons display's differently. Not sure why.

File: index.html [hunk 4]
```diff
@@ -272,26 +273,29 @@ <h2>Resources</h2>
...
+            </div>
+            <div class="resource">
+              <h3>Community Law</h3>
+              <h4>Free legal help throughout NZ.</h4>
+              <a class="btn btn-primary" a href="http://communitylaw.org.nz">Visit website</a>
```
Comment (anna-marie-vos) [inline index.html]: remove the "a" before href

File: index.html [hunk 5]
```diff
@@ -272,26 +273,29 @@ <h2>Resources</h2>
...
               <h2>Solo Parent Support</h2>
-            </br></br><h4>Being a solo parent can be challenging. Emotionally and financially. Make sure you have the support you need.</h4>
-              <a class="btn btn-primary" a href="http://www.cab.org.nz/Pages/home.aspx">Birthwise</a>
-              <a class="btn btn-primary" a href="http://www.cab.org.nz/Pages/home.aspx">Solo Mamas Aotearoa</a>
-              <a class="btn btn-primary" a href="http://www.cab.org.nz/Pages/home.aspx">Feminist Mothers Aotearoa</a>
-              <a class="btn btn-primary" a href="http://www.cab.org.nz/Pages/home.aspx">Women's Centres</a>
+              </br><h4>Being a solo parent can be challenging. Emotionally and financially. Make sure you have the support you need.</h4>
+              <a class="btn btn-primary" a href=" https://www.facebook.com/groups/346753365399969/">Solo Mamas Aotearoa</a>
+              <a class="btn btn-primary" a href="https://www.facebook.com/groups/feministmothersaotearoa/">Feminist Mothers Aotearoa</a>
+              <a class="btn btn-primary" a href="http://www.womenscentrerodney.org.nz/links.aspx">Women's Centres</a>
```
Comment (anna-marie-vos) [inline index.html]: Same thing here: remove the "a" before the href.

**Review by anna-marie-vos — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (janie-t) [reply index.html]: It should be a <div class="btn btn-primary><a></a></div>
Thats probs why the colours are different

File: index.html (hunk 1 above)
Comment (MeghanHughes) [reply index.html]: Good idea - done.

File: index.html (hunk 2 above)
Comment (MeghanHughes) [reply index.html]: Cool ok

Comment (MeghanHughes) [reply index.html]: Hey - All the links are news items should we use the news icon if they are all the same type? The law commission link up to the part where it talks about the Law Commission submission stuff (no point duplicating it here and also it throws out the symmetry!)

File: index.html (hunk 3 above)
Comment (MeghanHughes) [reply index.html]: Ah thanks Janie I was scratching my head why the links were different colours.
But Anna-Marie - I don't get why to remove the <a> tag? Then it won't link??

Comment (MeghanHughes) [reply index.html]: Hmmmm what have I done wrong here??
<div class="btn btn-primary" a href="https://www.familyservices.govt.nz/directory/">Visit website</a></div>
it is not linking to the href??

Comment (MeghanHughes) [reply index.html]: Sorry here's the code example:
```<div class="btn btn-primary" a href="https://www.familyservices.govt.nz/directory/">Visit website</a></div>```

Comment (janie-t) [reply index.html]: <div class="btn btn-primary><a href=""></a></div>
See how the a is in a separate tag

**Pull request merged**